pandas/_libs/tslibs/offsets.pyx

```diff
@@ -954,6 +963,27 @@ cdef class Tick(SingleConstructorOffset):
         return True
 
     def is_anchored(self) -> bool:
+        # GH#55388
+        """
+        Return False.
+
+        .. deprecated:: 2.2.0
+            is_anchored is deprecated and will be removed in a future version.
+            Use ``False`` instead.
+
+        Examples
+        --------
+        >>> pd.offsets.Hour().is_anchored()
+        False
+        >>> pd.offsets.Hour(2).is_anchored()
+        False
+        """
+        warnings.warn(
+            f"{type(self).__name__}.is_anchored is deprecated and will be removed "
```

Comment:
I don't understand this deprecation, how can I check for the equivalent attribute here? Use False is not really helpful

Reply:
for this class it's always `False`

for others you typically need to check that `obj.n==1` and `obj.weekday is None` (if it has a weekday attribute) and `obj.startingMonth is None` (if it has a `startingMonth` attribute)

what are you using it for?

Reply:
We are generally looking for freqs that aren’t anchored

I thought a little bit about this and the deprecations isn’t great, it makes those checks significantly more complex than before for users

Reply:
I thought that we can advise users use False, because `is_anchored` always returns `False` for `Tick` subclasses.
Do you think it would be better to use some expression that is always False instead of False, like `pd.offsets.Hour().n == None`?

Reply:
I mostly don’t want to have different check for every group, that’s what makes this ugly

Reply:
> that aren’t anchored

sure, but what do you mean by "not anchored"? Is it expected that `'1W-Wed'` is anchored, but `'2W-Wed'` isn't?

If so, and this function is useful to you and gives you exactly what you needed, then maybe it's OK to revert the deprecation

Reply:
> We are generally looking for freqs that aren’t anchored

what is your use case here?  `not obj.is_anchored()` is equivalent to `isinstance(obj, Tick) or obj.n != 1` which is just a weird thing to care about

Reply:
That helps, thx

We have to make a decision about shifting divisions, this basically goes back how the offset behaves on different values, e.g. if it can shift different values to the same target (if it's anchored for example) then we have to make a different decision compared to if every value maps to a distinct target value

Reply:
> e.g. if it can shift different values to the same target (if it's anchored for example)

An example might help here.  I _think_ what you're referring to is what past-me thought is_anchored meant (xref #44025), e.g "W-SUN" would be anchored, but "W" would not (it would act like "7D").  `pd.DateOffset(day=1)` would, but `pd.DateOffset(days=1)` would not.

NB: the "is equivalent to" i gave about isn't _quite_ right.

Reply:
```
idx = pd.Index([pd.Timestamp("2020-01-01"), pd.Timestamp("2020-01-02")])
print(idx.shift(1, freq=pd.tseries.offsets.Week(1)))
print(idx.shift(1, freq=pd.tseries.offsets.Week(1, weekday=2)))
```


```
DatetimeIndex(['2020-01-08', '2020-01-09'], dtype='datetime64[ns]', freq=None)
DatetimeIndex(['2020-01-08', '2020-01-08'], dtype='datetime64[ns]', freq=None)
```

The first one just shifts 7 days, so everything is fine


The second one shifts to a specific weekday though, which is an issue for us

Reply:
That matches what I think a more-useful is_anchored would mean.  My guess is we can come up with examples (e.g. the day=1 vs days=1 above) where checking is_anchored would be wrong for your use case.